### PR TITLE
update features.md

### PR DESCRIPTION
**Pull request by bowen12001**
### Related Issue

Solving Issue: #xxx

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [ ] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

File: content/introduction/features.md [hunk 1]
```diff
@@ -2,16 +2,52 @@
...
+**Lightweight & K8s native:**
+
+* One-click deployment, 10s to complete the installation.
+
+* No external dependencies, no need for zookeeper.
+* Limited resource usage, each pod in the cluster can run with 0.1 core and 500M memory.
+
+**Separation of storage and computing:**
+
+* The storage node is completely based on k8s design, multiple copies, and supports multi-level storage.
```
Comment (wenfengwang) [inline content/introduction/features.md:28]: multi-level -> multi-tier

File: content/introduction/features.md [hunk 2]
```diff
@@ -2,16 +2,52 @@
...
+* After the traffic is reduced, the cluster is reduced in seconds.
+
+* Both storage and compute clusters can be scaled up and down.
+
+**Lightweight & K8s native:**
+
+* One-click deployment, 10s to complete the installation.
+
+* No external dependencies, no need for zookeeper.
+* Limited resource usage, each pod in the cluster can run with 0.1 core and 500M memory.
```
Comment (wenfengwang) [inline content/introduction/features.md:24]: 0.5 core is better

File: content/introduction/features.md [hunk 3]
```diff
@@ -2,16 +2,52 @@
...
+
+**Separation of storage and computing:**
+
+* The storage node is completely based on k8s design, multiple copies, and supports multi-level storage.
+* There is an independent computing cluster that supports real-time computing of events, including filtering and conversion.
```
Comment (wenfengwang) [inline content/introduction/features.md:29]: conversion -> Transformation

**Review by wenfengwang — CHANGES_REQUESTED** [review]

File: content/introduction/features.md [hunk 4]
```diff
@@ -2,16 +2,52 @@
...
+
+* Provide event filtering and transformation capabilities.
+
+* Provide 100+ out-of-the-box event conversion functions.
+* Support custom event calculation based on cloud functions.
+* Support for window-based event computing (come soon).
+
+**Cloud function friendly:**
+
+* Real push model.
```
Comment (wenfengwang) [inline content/introduction/features.md:41]: model -> mode

File: content/introduction/features.md [hunk 5]
```diff
@@ -2,16 +2,52 @@
...
+* Native support for CloudEvents specification.
+
+* Seamlessly integrate cloud functions such as Aws lambda, Google Cloud Function, and Knative.
```
Comment (wenfengwang) [inline content/introduction/features.md:44]: AWS Lambda

**Review by wenfengwang — CHANGES_REQUESTED** [review]

File: content/introduction/features.md [hunk 6]
```diff
@@ -2,16 +2,52 @@
...
+
+* No need to manually scale up or down the cluster.
+
+* Expand new clusters in seconds after traffic increases.
+
+* After the traffic is reduced, the cluster is reduced in seconds.
+
+* Both storage and compute clusters can be scaled up and down.
+
+**Lightweight & K8s native:**
```
[the remark this answers is not in the document]
Comment (bowen12001) [reply content/introduction/features.md]: done

File: content/introduction/features.md (hunk 3 above)
Comment (bowen12001) [reply content/introduction/features.md]: done

File: content/introduction/features.md [hunk 7]
```diff
@@ -2,16 +2,52 @@
...
-- Configurable retry policy and dead letter queue.
-- Event retroactivity by time.
-- Arbitrary time second-precision delayed & scheduled events.
-- Unlimited partition retention capability beyond the size of single-machine storage.
-- Rapid, seamless scalability to add new machines.
+# Vanus Features
+
+Vanus has a vast amount of features to help users achieve their goals. Let us share with you some of the key features Vanus has to offer.
+
+**Entirely elastic:**
```
[the remark this answers is not in the document]
Comment (bowen12001) [reply content/introduction/features.md]: done

File: content/introduction/features.md (hunk 4 above)
Comment (bowen12001) [reply content/introduction/features.md]: done

File: content/introduction/features.md (hunk 5 above)
Comment (bowen12001) [reply content/introduction/features.md]: done

File: content/introduction/features.md [hunk 8]
```diff
@@ -2,16 +2,41 @@
...
-- Powerful event filtering and transformation.
-- Guaranteed at-least-once delivery.
-- Configurable retry policy and dead letter queue.
-- Event retroactivity by time.
-- Arbitrary time second-precision delayed & scheduled events.
-- Unlimited partition retention capability beyond the size of single-machine storage.
-- Rapid, seamless scalability to add new machines.
+# Vanus Features
+
+Vanus has a vast amount of features to help users achieve their goals. Let us share with you some of the key features Vanus has to offer.
```
Comment (Michaelg22) [inline content/introduction/features.md:7]:
```suggestion
Vanus offers an extensive range of features that help users accomplish their objectives. We'd like to highlight some of the key features that Vanus provides.
```

File: content/introduction/features.md [hunk 9]
```diff
@@ -2,16 +2,41 @@
...
+
+## Separation of storage and computing:
+* The storage node is completely based on k8s design, multiple copies, and supports multi-level storage.
+* There is an independent computing cluster that supports real-time computing of events, including filtering and Transformation.
+
+## Event computing:
+* Provide event filtering and transformation capabilities.
+* Provide 100+ out-of-the-box event conversion functions.
+* Support custom event calculation based on cloud functions.
+* Support for window-based event computing (come soon).
```
Comment (Michaelg22) [inline content/introduction/features.md:28]:
```suggestion
* Support for window-based event computing (coming soon).
```

**Review by Michaelg22 — CHANGES_REQUESTED** [review]

**Review by Michaelg22 — APPROVED** [review]

**Pull request merged**